tests/rule-engine/bc-modus-ponens.scm

```diff
@@ -0,0 +1,56 @@
+; =============================================================================
+; Modus Ponens Rule
+;
+; Given P(A implies B) and sA, calculate sB
+; -----------------------------------------------------------------------------
+
+(define pln-rule-modus-ponens
+    (BindLink
+        (VariableList
+            (VariableNode "$A")
+            (VariableNode "$B"))
+        (ImplicationLink
+            (AndLink
+                (ImplicationLink
+                    (VariableNode "$A")
+                    (VariableNode "$B"))
+                (VariableNode "$A"))
+            (ExecutionOutputLink
+                (GroundedSchemaNode "scm: pln-formula-simple-modus-ponens")
+                (ListLink
+                    (VariableNode "$B")
+                    (ImplicationLink
+                        (VariableNode "$A")
+                        (VariableNode "$B")))))))
+
```

Comment:
@williampma just in case you didn't know, I put some example of inference system using the URE under

examples/rule-engine

it is crisp, but it could have been re-used instead of redefining rules here.

No problem if you're aware of that and you prefer to redefine test rules here, just wanted to be sure you know about this examples directory.